practice/032/ex.ml

```diff
@@ -0,0 +1,15 @@
+open OUnit2
+
+module type Testable = sig
+  val gcd : int -> int -> int
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let v = "gcd" >::: [
+    "nil" >:: (fun _ -> assert_equal 13 (Tested.gcd 13 0));
+    "cons" >:: (fun _ -> assert_equal 2 (Tested.gcd 20536 7826));
```

Comment:
Change the name of the tests into `gcd_13_0` and `gcd_20536_7826`

Reply:
Could you take a look at it, is this what ask me to do?